CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 3.7.0 (2019-11-15)
+
+- Added `tag` input to the repeater item component and `orderChange` output to the repeater component to allow tracking user updates to the repeater item order. [#91](https://github.com/blackbaud/skyux-lists/pull/91)
+- Fixed the repeater component to no longer throw console errors when a reorderable repeater has no items. [#90](https://github.com/blackbaud/skyux-lists/pull/90) (Thanks [@blackbaud-GavinNicol](https://github.com/blackbaud-GavinNicol)!)
```

Comment:
"Fixed the repeater component to properly handle reorderable repeaters without items."

I could also be swayed to keep the original though but add a comma after "Thanks." :)

Reply:
done!